lib/collective.lib.js

```diff
@@ -49,6 +49,15 @@ export const getCollectiveMainTag = (hostCollectiveId = null, tags = [], type, s
   return CollectiveCategory.COLLECTIVE;
 };
 
+export const expenseSubmissionAllowed = (collective, user) => {
+  if (!collective?.settings.disablePublicExpenseSubmission) {
```

Comment:
Small protection since `settings` is nullable

```suggestion
  if (!collective?.settings?.disablePublicExpenseSubmission) {
```

Reply:
Thanks and done. 👍🏽